src/main/java/com/ait/lienzo/client/core/image/ImageLoader.java

```diff
@@ -71,7 +86,53 @@ public final void onError(final ErrorEvent event)
         }
     }
 
-    private final void doImageElementLoadAndRetry(final ImageElement elem, final Image image, final String orig, final String url)
+    public ImageLoader(final ImageResource resource)
+    {
+        this(resource,
+             new Image());
+    }
+
+    public ImageLoader(final ImageResource resource,
+                       final Image image)
+    {
+        final ImageElement element = ImageElement.as(image.getElement());
+
+        image.setVisible(false);
+
+        final HandlerRegistrationManager m_HandlerRegManager = new HandlerRegistrationManager();
+
+        m_HandlerRegManager.register(
```

Comment:
Some crazy formatting here.. 

I know ```ahome-it``` doesn't follow ```KIE``` but formatting is probably not consistent with ```ahome-it``` either.